subworkflows/local/parse_input.nf

```diff
@@ -38,12 +38,13 @@ def parse_samplesheet(LinkedHashMap row, single_end) {
 workflow PARSE_INPUT {
     take:
     input // file.tsv or folder
+    is_fast_input
     single_end
     multiple_sequencing_runs
     extension
 
     main:
-    if ( input.toString().toLowerCase().endsWith(".fasta") || input.toString().toLowerCase().endsWith(".fna") || input.toString().toLowerCase().endsWith(".fa") ) {
+    if ( is_fast_input ) {
```

Comment:
Should that be `is_fasta_input`? Also above.

Reply:
Yes, fixed!